CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 * Fix some out-of-bounds reports from LGTM
 * Update asm to 7.0 for better Java 11 support ([#785](https://github.com/spotbugs/spotbugs/pull/785))
 * Ignore @FXML annotated fields in UR\_UNIT\_READ ([#702](https://github.com/spotbugs/spotbugs/issues/702))
+* Add new detector IRA\_INEFFICIENT\_REPLACEALL for detecting usage of String.replaceAll where no regex is being used ([#705](https://github.com/spotbugs/spotbugs/issues/705))
```

Comment:
This is not bugfix, so please use `Added` instead of `Fixed`. refer https://keepachangelog.com/en/1.0.0/ for detail

Reply:
This actually says "Add". Should it be "Added"? I just looked at row 10 for an example. I can change all of the lines use that if that is the preferred form.

Reply:
@lasselindqvist I think he means that it's in the wrong section of the changelog. It's filed under `### Fixed` instead of `### Added`.

Reply:
Done. I will squash the changes together later today.